src/sparseml/transformers/text_classification.py

```diff
@@ -35,6 +35,7 @@
 import numpy as np
 import transformers
 from datasets import load_dataset, load_metric
+from sklearn.model_selection import StratifiedShuffleSplit
```

Comment:
Is `sklearn` installed as a part of our transformers fork? if not could we add it to `setup.py`

Reply:
Added scikit-learn to setup.py